fuzz_lightyear/datastore.py

```diff
@@ -10,6 +11,57 @@
 from typing import Tuple
 
 
+# These are module variables which contain the post-fuzz hooks
+# which have been registered. Each global allows fuzz_lightyear
+# to get a list applicable to a certain operation or tag.
+_POST_FUZZ_HOOKS_BY_OPERATION = defaultdict(set)  # type: Dict[str, Set[Callable]]
+_POST_FUZZ_HOOKS_BY_TAG = defaultdict(set)  # type: Dict[str, Set[Callable]]
+
+
+def register_post_fuzz_hook(
+    hook: Callable[[Dict[str, Any]], Dict[str, Any]],
+    operation_ids: Optional[List[str]] = None,
+    tags: Optional[List[str]] = None,
+) -> None:
+    """Adds a post-fuzz hook to fuzz_lightyear's store of post-fuzz
+    hooks.
+
+    :param hook: The hook.
+    :param operation_ids: A list of operations that the input hook
+    applies to.
+    :param tags: A list of Swagger tags that the input hook applies
+    to.
+    """
+    if not operation_ids and not tags:
+        operation_ids = ['*']
+
+    if not operation_ids:
+        operation_ids = []
+
+    if not tags:
+        tags = []
```

Comment:
Will `mypy` get angry if the default value for `operation_ids` and `tags` is set to `[]` empty list instead of `None`? These lines could be omitted.